frontend/public/components/dashboards-page/overview-dashboard/details-card.tsx

```diff
@@ -111,7 +118,15 @@ export const DetailsCard_ = connect(mapStateToProps)(({
                 error={!openShiftVersion}
                 isLoading={!clusterVersionLoaded && !clusterVersionError}
               >
-                {openShiftVersion}
+                <Link to="/settings/cluster/">{getOpenShiftVersion(clusterVersionData)}</Link>
+              </DetailItem>
+              <DetailItem
+                key="updatechannel"
+                title="Update channel"
+                isLoading={!clusterVersionLoaded && !clusterVersionError}
+                error={!!clusterVersionError}
```

Comment:
I'd add `|| !getCVChannel(clusterVersionData)`

after adding it, can you extract the `getCVChannel(clusterVersionData)` so we call it just once for error and children ?

Reply:
done